qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -901,6 +902,13 @@ public void testSnapshotRestore() throws IOException {
         templateBuilder.endObject().endObject();
         Request createTemplateRequest = new Request("PUT", "/_template/test_template");
         createTemplateRequest.setJsonEntity(Strings.toString(templateBuilder));
+
+        // In 7.0, type names are no longer expected by default in put index template requests.
+        // We therefore use the deprecated typed APIs when running against the current version.
+        if (!isRunningAgainstOldCluster()) {
```

Comment:
nit: (isRunningAgainstOldCluster() == false). I know it's a habit thats hard to break ;-) Only change this if you have to do other changes anyway.

Reply:
Yes, it might take me a while to retrain my fingers!